From 76e35021d9301e7efcccb9b9dbacdba7e71a3bca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert=20G=C3=BCnzler?= <r@gnzler.io>
Date: Fri, 22 Oct 2021 15:36:54 +0200
Subject: [PATCH 1/5] homedir fix

---
 src/musikcore/support/Common.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/musikcore/support/Common.cpp b/src/musikcore/support/Common.cpp
index 58bc4d2..79120ef 100644
--- a/src/musikcore/support/Common.cpp
+++ b/src/musikcore/support/Common.cpp
@@ -82,11 +82,7 @@ static inline void silentDelete(const std::string fn) {
 namespace musik { namespace core {
 
     static std::string getDataDirectoryRoot() {
-    #ifdef WIN32
         return GetHomeDirectory();
-    #else
-        return GetHomeDirectory() + "/.config";
-    #endif
     }
 
     std::string GetPluginDirectory() {
-- 
2.32.0

