From db277851c4ebce35c4ca8e5f15d71119dec9d2d4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert=20G=C3=BCnzler?= <r@gnzler.io>
Date: Tue, 29 Mar 2022 12:22:36 +0200
Subject: [PATCH] Explicitely require linking to libtinfow

---
 src/musikcube/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/musikcube/CMakeLists.txt b/src/musikcube/CMakeLists.txt
index f42748a..daf9512 100644
--- a/src/musikcube/CMakeLists.txt
+++ b/src/musikcube/CMakeLists.txt
@@ -91,10 +91,12 @@ if (${NCURSES_DISABLE_LIB_SUFFIXES} MATCHES "true")
     message(STATUS "[ncurses] using library names *WITHOUT* 'w' prefix")
     set(CURSES_LIBRARY_NAME ncurses)
     set(PANEL_LIBRARY_NAME panel)
+    set(TINFO_LIBRARY_NAME tinfo)
 else()
     message(STATUS "[ncurses] using library names with 'w' prefix")
     set(CURSES_LIBRARY_NAME ncursesw)
     set(PANEL_LIBRARY_NAME panelw)
+    set(TINFO_LIBRARY_NAME tinfow)
 endif()
 
 if (APPLE)
@@ -103,7 +105,7 @@ if (APPLE)
     set(PANEL_LIBRARY_NAME "lib${PANEL_LIBRARY_NAME}.a")
 else()
     message(STATUS "[ncurses] not Darwin! will attempt to link against libtinfo")
-    find_library(LIBTINFO NAMES tinfo)
+    find_library(LIBTINFO NAMES ${TINFO_LIBRARY_NAME})
     message(STATUS "[musikcube] using libtinfo at: " ${LIBTINFO})
 endif()
 
-- 
2.34.1

