From 91e7ab5b90dbd42cd3508e60670c61750826a512 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert=20G=C3=BCnzler?= <r@gnzler.io>
Date: Fri, 22 Oct 2021 15:38:04 +0200
Subject: [PATCH 2/5] cmake: allow disabling pulse and alsa

---
 CMakeLists.txt | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 964d59f..907917a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -174,8 +174,12 @@ add_dependencies(musikcube musikcore taglibreader nullout server httpdatastream
 add_dependencies(musikcubed musikcube)
 
 if (CMAKE_SYSTEM_NAME MATCHES "Linux")
-  add_subdirectory(src/plugins/alsaout)
-  add_subdirectory(src/plugins/pulseout)
+  if (${ENABLE_ALSA} MATCHES "true")
+    add_subdirectory(src/plugins/alsaout)
+  endif()
+  if (${ENABLE_PULSEAUDIO} MATCHES "true")
+    add_subdirectory(src/plugins/pulseout)
+  endif()
   if (${ENABLE_PIPEWIRE} MATCHES "true")
     add_subdirectory(src/plugins/pipewireout)
     add_dependencies(musikcube pipewireout)
-- 
2.32.0

