From 97dab1b970243a47a02ba57963644a6622013f19 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert=20G=C3=BCnzler?= <r@gnzler.io>
Date: Wed, 27 Oct 2021 13:02:23 +0200
Subject: [PATCH 5/5] cmake: disable gmedecoder

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 16c349d..1b76717 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -130,7 +130,7 @@ add_subdirectory(src/plugins/nullout)
 add_subdirectory(src/plugins/httpdatastream)
 add_subdirectory(src/plugins/stockencoders)
 add_subdirectory(src/plugins/supereqdsp)
-add_subdirectory(src/plugins/gmedecoder)
+# add_subdirectory(src/plugins/gmedecoder)
 
 find_library(LIBOPENMPT NAMES openmpt)
 if ("${LIBOPENMPT}" STREQUAL "LIBOPENMPT-NOTFOUND")
-- 
2.32.0

