From 2621daf3150a3f7ff1f87e181421b9f3ca92e9fe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert=20G=C3=BCnzler?= <r@gnzler.io>
Date: Tue, 28 Jan 2020 18:49:36 +0100
Subject: [PATCH 6/6] Allow overwriting version

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3f06c4d..11071bf 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 .SUFFIXES:
 .SUFFIXES: .1 .5 .7 .1.scd .5.scd .7.scd
 
-VERSION=0.3.0
+VERSION?=0.3.0
 
 VPATH=doc
 PREFIX?=/usr/local
-- 
2.24.1

